docs/content/en/docs/overview/_index.md

```diff
@@ -4,10 +4,10 @@ linkTitle: "Overview"
 weight: 1
 icon: "overview"
 description: >
-  Discover Tetragon and its capabilities
+  Discover Cilium's Tetragon and its capabilities
```

Comment:
```suggestion
  Discover Cilium Tetragon and its capabilities
```

Reply:
@lizrice can you clarify this? i was a bit confused, how do we want to position this? is it Cilium's tetragon or Cilium tetragon?

Reply:
"Cilium Tetragon" is correct, so Michi's suggested changes are good

Reply:
oh thanks, I was a bit confused on if we are now calling it Cilium Tetragon or if we want to indicate that it's a part of Cilium by just saying Cilium's Tetragon.

@michi-covalent i've committed the change